src/main/java/it/gov/pagopa/cgn/portal/model/ProfileEntity.java

```diff
@@ -148,6 +150,11 @@ public class ProfileEntity extends BaseEntity {
     @Column(name = "support_value", length = 500)
     private String supportValue;
 
+    @Enumerated(EnumType.STRING)
+    @Column(name = "entity_type", length = 24)
+    @NotNull
+    private EntityTypeEnum entityType;
```

Comment:
I don't get the reason to have it also on profile...

Reply:
removed